.github/workflows/release.yml

```diff
@@ -1,86 +1,103 @@
-              ./target/release/forge
-              ./target/release/cast
+      - name: Strip binaries (non-Windows)
+        if: matrix.job.platform != 'win32'
+        run: |
+          cd target/release
+          strip cast
+          strip forge
 
-  # creates a GH release with the built binaries and CHANGELOG entry
-  create-release:
-    runs-on: ubuntu-latest
-    needs:  build-artifacts
-    steps:
-     - name: Checkout
-       uses: actions/checkout@v2
-       with:
-        fetch-depth: 1000
-
-     - name: Set output
-       id: vars
-       run: echo ::set-output name=tag::${GITHUB_REF#refs/*/}
-
-     - name: Restore artifacts
-       uses: actions/download-artifact@v2
-
-     - name: Build Changelog
-       id: github_release
-       uses: mikepenz/release-changelog-builder-action@v2.4.2
-       env:
-        GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
-
-     - name: Create Release
-       id: create-release
-       uses: softprops/action-gh-release@v0.1.13
-       env:
-        GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
-       with:
-         tag_name:  ${{ steps.vars.outputs.tag }}
-         release_name: ${{ github.ref }}
-         body: ${{steps.build_changelog.outputs.changelog}}
-         files: |
-           ./target-macos-latest
-           ./target-ubuntu-latest
+      - name: Archive binaries
+        id: artifacts
+        env:
+          PLATFORM_NAME: ${{ matrix.job.platform }}
+          ARCH: ${{ matrix.job.arch }}
+        run: |
+          if [ "$PLATFORM_NAME" != "win32" ]; then
+            tar -czvf "foundry_${TAG_NAME}_${PLATFORM_NAME}_${ARCH}.tar.gz" -C ./target/release forge cast
+            echo "::set-output name=file_name::foundry_${TAG_NAME}_${PLATFORM_NAME}_${ARCH}.tar.gz"
+          else
+            cd ./target/release
+            7z a -tzip "foundry_${TAG_NAME}_${PLATFORM_NAME}_${ARCH}.zip" forge.exe cast.exe
+            mv "foundry_${TAG_NAME}_${PLATFORM_NAME}_${ARCH}.zip" ../../
+            echo "::set-output name=file_name::foundry_${TAG_NAME}_${PLATFORM_NAME}_${ARCH}.zip"
+          fi
+        shell: bash
+ 
+      - name: Build changelog
+        id: build_changelog
+        if: ${{ env.IS_NIGHTLY == false }}
+        uses: mikepenz/release-changelog-builder-action@v2.4.2
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-  # after the GH release is done, proceed to open a PR on homebrew-core
-  publish-homebrew:
-    runs-on: ubuntu-latest
-    needs:  build-artifacts
-    steps:
-      - name: Create homebrew-core PR
-        uses: mislav/bump-homebrew-formula-action@v1.12
+      - name: Create release
+        uses: softprops/action-gh-release@v1
         with:
-          formula-name: foundry
-        env:
-          COMMITTER_TOKEN: ${{ secrets.GH_TOKEN }}
+          tag_name: ${{ env.TAG_NAME }}
+          draft: ${{ env.IS_NIGHTLY }}
+          body: ${{ steps.build_changelog.outputs.changelog }}
+          files: ${{ steps.artifacts.outputs.file_name }}
```

Comment:
We can add a `token` here to set the committer's identity, but that requires a personal access token. See [here](https://github.com/softprops/action-gh-release#-external-release-notes).

@gakonst let me know if that is something you want to do, then I'll add it (just tell me the name of the secret)